core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1607,6 +1708,22 @@ class TopicChangeListener(controller: KafkaController, eventManager: ControllerE
   }
 }
 
+/**
+  * Called when broker notifies controller of disk change
+  *
+  * @param controller
+  */
+class LogDirEventNotificationListener(controller: KafkaController, eventManager: ControllerEventManager) extends IZkChildListener with Logging {
```

Comment:
I agree. But all existing listener class (e.g. PartitionReassignmentListener) takes this as the second argument. It seems better to keep the same code style for `LogDirEventNotificationListener`, or we can update all of them to remove the second argument `eventManager`. Which solution do you prefer?